src/main/kotlin/com/openlattice/organizations/DatasetController.kt

```diff
@@ -89,7 +92,7 @@ class DatasetController : DatasetApi, AuthorizingComponent {
     override fun getExternalDatabaseTableWithColumns(
             @PathVariable(ID) organizationId: UUID,
             @PathVariable(TABLE_ID) tableId: UUID): OrganizationExternalDatabaseTableColumnsPair {
-        ensureReadAccess(AclKey(organizationId, tableId))
+        ensureReadAccess(AclKey(tableId))
```

Comment:
How come the acl key does not contain the organizationId anymore?

Reply:
This allows audit logs will be per table rather than per organization. Audit logs correspond to the root of the acl key, so with the org id as the root, all audit logs would be in the org audit log.

Reply:
I see, thanks for the explanation!